overops-functions/src/main/java/com/takipi/udf/microsoftteams/card/MicrosoftPotentialAction.java

```diff
@@ -0,0 +1,56 @@
+package com.takipi.udf.microsoftteams.card;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class MicrosoftPotentialAction {
+    @SerializedName("@type")
+    public String type;
```

Comment:
Members of this class can probably be `final`

Reply:
done